tests/model_forms/models.py

```diff
@@ -356,6 +356,9 @@ def __iter__(self):
     def __str__(self):
         return self.name
 
+    class Meta:
+        ordering = ('name',)
+
```

Comment:
Is this change related?

Reply:
I don't remember. Works fine without it now. I'll remove it.